### PR TITLE
New invitation flow

### PR DESCRIPTION
**Pull request by gowsik-ragunath**
## Summary

Replace the devise invitatable with custom invitation flow.

## Preview

https://www.loom.com/share/478ef54739c54b85a5abcfe2e54cf3cf

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [X] I have manually tested all workflows
- [X] I have performed a self-review of my own code
- [X] I have added automated tests for my code

### CODE REVIEW

File: app/views/mailers/user_invitation_mailer/send_user_invitation.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type'/>
+  </head>
+  <body style="color:#1D1A31">
+  <p>Hi <%= @name %>, </p>
+
+  <p>
+    We are so delighted to have you onboard. We want to let you know some of the best ways to get up and running on our portal -
```
Comment (akhilgkrishnan) [inline app/views/mailers/user_invitation_mailer/send_user_invitation.html.erb:10]: @supriya3105 can you verify this email content.

**Review by akhilgkrishnan — COMMENTED** [review]
@gowsik-ragunath I've one concern, where we are storing the existing user's invitation data. Do we have a data migration for moving the existing data to the newer table?

File: app/views/mailers/user_invitation_mailer/send_user_invitation.html.erb (hunk 1 above)
Comment (supriya3105) [reply app/views/mailers/user_invitation_mailer/send_user_invitation.html.erb:10]: The content is as below

Comment (apoorv1316) [reply app/views/mailers/user_invitation_mailer/send_user_invitation.html.erb:10]: The Content is a little bit different from what is in the document. Also, we are using 2 different email content for when we invite a user and when a owner confirms his email. There is no content for "Confirmation Instructions" in the docs provided @supriya3105 @gowsik-ragunath

**Review by apoorv1316 — APPROVED** [review]
Looks good to me

**Comment by harshaanNihal** [comment]
@supriya3105 @gowsik-ragunath Merged develop to resolve conflicts.

**Pull request merged**